pom.xml

```diff
@@ -2962,6 +2962,21 @@
         </dependencies>
     </dependencyManagement>
 
+    <pluginRepositories>
+        <pluginRepository>
+            <releases>
+                <enabled>false</enabled>
+            </releases>
+            <snapshots>
+                <enabled>true</enabled>
+            </snapshots>
+            <id>apache.snapshots</id>
```

Comment:
this doesn't seem related to upgrading mockito ... and I may be naive but we pull snapshot versions of plugins ? 
doesn't that make our build non reproductible since snapshot versions may be updated completely randomly ?

Reply:
Yes related to some tries on JPA updades +1 shall be removed...